### ISSUE TITLE
Ambiguous messages after running "tns doctor" after installing

### ISSUE THREAD

**Issue by saewoonam** [NativeScript/nativescript-cli#5331]
I am trying to do a fresh install of nativecript on a OS X Catalina Mac laptop.   After running the CLI installer, I ran "tns doctor" to check the installation.   I got the following message:

$tns doctor                                                                                                                      
✔ Getting environment information 

No issues were detected.
✔ Your ANDROID_HOME environment variable is set and points to correct directory.
✔ Your adb from the Android SDK is correctly installed.
✔ The Android SDK is installed.
✔ A compatible Android SDK for compilation is found.
✔ Javac is installed and is configured properly.
✔ The Java Development Kit (JDK) is installed and is configured properly.
✔ Xcode is installed and is configured properly.
✔ xcodeproj is installed and is configured properly.
✔ CocoaPods are installed.
✔ CocoaPods update is not required.
✔ CocoaPods are configured properly.
✔ Your current CocoaPods version is newer than 1.0.0.
✔ Python installed and configured correctly.
✔ The Python 'six' package is found.
✔ Xcode version 11.5.0 satisfies minimum required version 10.
✔ Getting NativeScript components versions information...
✔ Component nativescript has 6.7.4 version and is up to date.
unable to resolve processService

**Describe the bug**
In my terminal the last line appears in red... Indicating that there is a problem.   I am not sure whether the installation is completely successful... The initial message state the "No issues were detected"... However, it seems to have missed something.

**Comment by saewoonam** [comment]
The installation appeared to have failed.   I tried to run the HelloWorld project.. After compiling I get the following response:

~/D/p/c/HelloWorld >tns preview                                                                                                                                   
unable to resolve processService
~/D/p/c/HelloWorld >

**Comment by NathanWalker** [comment]
@saewoonam could you try:
```
npm uninstall -g nativescript
```
then
```
npm install -g nativescript
```
which should give you 6.7.6

**Comment by saewoonam** [comment]
I tried uninstall / install multiple times for different version of node (10, 12, and 14). 
none of those things worked.
I uninstalled node and re-installed node  and nvm...  Also, did not help...
I eventually completely erased all traces of node from my Mac (including default version with OS)
and re-installed node with brew... Not happy I did this BTW.
And, it now works.   Not sure what the problem was with node.
